beancount_import/source/schwab_csv.py

```diff
@@ -346,6 +378,31 @@ def get_processed_entry(
                 fees=self.fees,
                 **shared_attrs,
             )
+        if self.action is BrokerageAction.EXPIRED:
+            assert self.quantity is not None
+            amount = Amount(Decimal(0), currency=CASH_CURRENCY) if self.amount is None else self.amount
```

Comment:
I think the wrong version of this block was kept after merge conflict, and we've now lost some of my earlier feedback. Notably we are redefining `amount` here after already having defined it above, and we are not using `**shared_attrs` below.

Reply:
Good catch.